CMakeLists.txt

```diff
@@ -426,6 +426,26 @@ if(NOT WIN32)
 endif()
 
 find_package(Readline REQUIRED)
+find_package(Augeas REQUIRED)
+
+if(LINUX)
+  find_package(LibSELinux REQUIRED)
+endif()
+
+if(APPLE)
+  if(NOT DEFINED ENV{BUILD_LINK_SHARED})
```

Comment:
Hmm, I'm not sure if this logic is needed. Do you expect the `libxml2` package to be "unlinked" in brew's terminology?